backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -555,19 +555,18 @@ public async System.Threading.Tasks.Task<Project> UpdateAsync(Project project)
         /// <exception cref="KeyNotFoundException">Project does not exist.</exception>
         /// <exception cref="NotAuthorizedException">User does not have permission to delete project.</exception>
         /// <returns></returns>
-        public async System.Threading.Tasks.Task RemoveAsync(Project project)
+        public async System.Threading.Tasks.Task<Project> RemoveAsync(Project project)
         {
             project.ThrowIfNotAllowedToEdit(nameof(project), this.User, new[] { Permissions.ProjectDelete, Permissions.AdminProjects });
 
             var userAgencies = this.User.GetAgencies();
             var isAdmin = this.User.HasPermission(Permissions.AdminProjects);
             var originalProject = this.Context.Projects
                 .Include(p => p.Status)
-                .Include(p => p.Properties)
-                .ThenInclude(p => p.Parcel)
+                .Include(p => p.Properties).ThenInclude(p => p.Parcel).ThenInclude(p => p.Parcels)
```

Comment:
Why do you need to include `Parcels`?  Where you didn't need to include `Buildings` below?

Reply:
this is to get the parent parcels of subdivisions, not relevant for buildings.